src/modules/auth/auth.service.ts

```
@@ -0,0 +1,289 @@
+
+    const accessToken = this.jwtService.sign(payload, {
+      expiresIn: this.configService.jwtAccessTokenExpiration,
+    });
+
+    const refreshTokenValue = uuidv4(); // Token brut à retourner au client
+    const refreshTokenHash = this.hashToken(refreshTokenValue);
+
+    const refreshExpiresIn = new Date();
+    refreshExpiresIn.setDate(
+      refreshExpiresIn.getDate() +
+        parseInt(this.configService.jwtRefrehTokenExpiration?.replace('d', '') || '7', 10),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)